x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -123,13 +127,32 @@ export function DiagnosticsApmDocuments() {
       {isImported && diagnosticsBundle ? (
         <>
           <EuiBadge>
-            From: {new Date(diagnosticsBundle.params.start).toISOString()}
+            {i18n.translate(
+              'xpack.apm.diagnosticsApmDocuments.from:BadgeLabel',
+              { defaultMessage: 'From:' }
+            )}
+            {new Date(diagnosticsBundle.params.start).toISOString()}
```

Comment:
What do you think of:
```suggestion
            {i18n.translate(
              'xpack.apm.diagnosticsApmDocuments.from:BadgeLabel',
              { defaultMessage: 'From: {date}',
              values: {
                date: new Date(diagnosticsBundle.params.start).toISOString()
              } }
            )}
```